config.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: cpp
+script: make test
```

Comment:
File looks reasonable! (clang is not supported in makefile anymore but thats fine) not sure but could this also make run-coverage or is this just to build? make test doesnt rull anything atm i think

ohh but now that i think of it maybe it would falsify the integration results so maybe not